libweasyl/libweasyl/staff.py

```diff
@@ -1,10 +1,11 @@
 """
 Dictionaries of Weasyl staff.
 
-This module must be initialized with a call to `staff._init_staff()`
-with the path to a yaml config file where staff members are identified
-by their user id. Example contents of such a config file might look as
-follows:
+This module must be initialized with a call to `staff._init_staff()`,
+passing in a dict of staff members. Normally, this dict is generated by
```

Comment:
Not sure that libweasyl documentation needs to be concerned with weasyl implementation details. But again, that opinion might not be shared.